docs/index.md

```diff
@@ -429,6 +429,31 @@ key: 'value'
 key: "value"
 ```
 
+### Preserve Quotes
+
+Default: `preserve_quotes: bool = False`<br>
+Environment variable override:
+```bash
+export YAMLFIX_preserve_quotes="false"
+```
+
+Keep quotes as they are
+
+```yaml
+# original
+key: [value, 'value with spaces']
+```
+
+```yaml
+# Option set to: false
+key: [value, value with spaces]
+```
+
+```yaml
+# Option set to: true
+key: [exec, 'value with spaces']
```

Comment:
Shouldn't this line be?

```yaml
key: [value, 'value with spaces']
```

So `exec` -> `value`.

Reply:
I fixed this typo, thanks 🙇🏼